packages/contracts/migrations/13_create_super_group.ts

```diff
@@ -0,0 +1,34 @@
+/* global artifacts */
+import { prepareForceUnionMessage, promisify } from "@joincivil/utils";
+import { POU_SUPER_GROUP } from "../test/utils/constants";
+
+module.exports = (deployer: any, network: string, accounts: string[]) => {
+  const [OPERATOR] = accounts;
+  const signAsync = promisify<string>(web3.eth.sign, web3.eth);
+
+  const CivilTCR = artifacts.require("CivilTCR");
+  const Parameterizer = artifacts.require("CivilParameterizer");
+  const PLCRVoting = artifacts.require("CivilPLCRVoting");
+  const Government = artifacts.require("Government");
```

Comment:
I think we could skip adding the government to the supergroup, it doesn't accept tokens

Reply:
Great, will do